data/campaigns/tutorial/scenarios/02_Tutorial_part_2.cfg

```diff
@@ -869,6 +869,24 @@ A full list of abilities and weapons specials, along with traits, may be found i
         [/filter_condition]
 
         {TALK_ABOUT Thrag ( _ "You are close to killing their leader! The unit who finishes him will gain 16 experience points because he is second level. Choose your attacking unit carefully!")}
+
+        [message]
+            speaker=student
+            message= _ "Should I choose a unit that will level up?"
+            message_female= _ "female^Should I choose a unit that will level up?"
+        [/message]
+
+        {TALK_ABOUT Thrag ( _ "That’s a difficult question. Consider which advancements will strengthen your army, and whether the unit should advance now or later.")}
+
+        {TALK_ABOUT Thrag ( _ "Some second level units strengthen your army far more than just having a stronger version of the first level unit. A Captain’s leadership makes every level 1 unit do 25% more damage when he’s next to them. A Druid heals 8 hitpoints instead of 4, and her ability to cure poison is invaluable when you are facing enemies that use it. The powerful, accurate attacks of a Sorceress or Marksman are good for dislodging fortified enemy units. Having these units already levelled on your recall list can make a huge difference in the next battle.")}
+
+        {TALK_ABOUT Thrag ( _ "If your army already has a few units with those abilities, then I’d normally choose a unit that needs 17-24 experience points to advance, so that they’ll be about to level. Units are fully healed when they advance a level, and all units are fully healed between scenarios. Also, until they level, they’ll cost less gold per turn in upkeep.")}
```

Comment:
```suggestion
        {TALK_ABOUT Thrag ( _ "If your army already has a few units with those abilities, I would normally choose a unit that needs 17-24 experience points to advance so that they will be close to leveling in the following scenario. All units are fully healed between scenarios and are also fully healed when they advance a level. You can use this to replenish a unit's health in difficult situations. A lower level unit also requires less gold per turn in upkeep.")}
```

Reply:
Thanks, will apply this.

Reply:
I'm not so sure about this part.

1. With the comma before "so" removed, I find the sentence harder to parse.

2. The last sentence doesn't flow as well from the previous ones as it did before.

3. The last three sentences feel like a wall of text. They don't connect with each other well, and they're written as though spoken by the narrator, not by Galdrad. Maybe add some connectives? Or split the line into two messages? Or remove some information outright (some of it has already been said, or just leave it in the help only)? Or move some information to another event (there's a message about advancement in the first scenario that might be possible to extend)? Or add a new scenario to the tutorial where the player is given, say, an Elvish Fighter with 1HP and 39/40XP against a Dark Adept? We could create similar setups to teach about leadership, etc (there's a scenario in Inky's puzzles addon that we could draw inspiration from).